crowbar_framework/app/models/node_object.rb

```diff
@@ -1360,7 +1360,7 @@ def disk_release(device, owner)
     if device
       crowbar_wall[:claimed_disks] ||= {}
 
-      unless disk_owner(device) == owner
+      if owner.empty? || disk_owner(device) != owner
```

Comment:
Ah, I read wrong. I guess the issue is that this is called with an empty owner -- this should not happen, so it would be a bug in the code calling this method. Unless we allow forcing a release that way (?).

Do you have more details on when this is happening?